src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -18,25 +19,25 @@
  */
 public class SampleDataUtil {
     public static Person[] getSamplePersons() {
-        return new Person[] {
+        return new Person[]{
```

Comment:
Perhaps we should add back the space here?

Reply:
Also not detected.